datasets/sem_eval_2014_task_1/sem_eval_2014_task_1.py

```diff
@@ -0,0 +1,130 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The SemEval-2014 Task 1 on Evaluation of Compositional Distributional Semantic Models on Full Sentences through Semantic Relatedness and Entailment"""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{inproceedings,
+author = {Marelli, Marco and Bentivogli, Luisa and Baroni, Marco and Bernardi, Raffaella and Menini, Stefano and Zamparelli, Roberto},
+year = {2014},
+month = {08},
+pages = {},
+title = {SemEval-2014 Task 1: Evaluation of Compositional Distributional Semantic Models on Full Sentences through Semantic Relatedness and Textual Entailment},
+doi = {10.3115/v1/S14-2001}
+}
+"""
+
+_DESCRIPTION = """\
+The SemEval-2014 Task 1 focuses on Evaluation of Compositional Distributional Semantic Models
+on Full Sentences through Semantic Relatedness and Entailment. The task was designed to
+predict the degree of relatedness between two sentences and to detect the entailment
+relation holding between them.
+"""
+
+_HOMEPAGE = "https://alt.qcri.org/semeval2014/task1/"
+
+_URLs = {
+    "train": "https://alt.qcri.org/semeval2014/task1/data/uploads/sick_train.zip",
+    "test": "https://alt.qcri.org/semeval2014/task1/data/uploads/sick_test_annotated.zip",
+    "validation": "https://alt.qcri.org/semeval2014/task1/data/uploads/sick_trial.zip",
+}
+
+
+class SemEval2014Task1(datasets.GeneratorBasedBuilder):
+    """The SemEval-2014 Task 1 on Evaluation of Compositional
+    Distributional Semantic Models on Full Sentences through
+    Semantic Relatedness and Entailment
+    """
+
+    VERSION = datasets.Version("1.0.0")
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # This defines the different columns of the dataset and their types
+            features=datasets.Features(
+                {
+                    "sentence_pair_id": datasets.Value("int64"),
+                    "sentence_A": datasets.Value("string"),
+                    "sentence_B": datasets.Value("string"),
```

Comment:
could you rename them `premise` and `hypothesis` ? that's the standard we use for NLI datasets

Reply:
Fixed this. Didn't know about the standard. My bad!